src/lib/wait.c

```diff
@@ -64,3 +64,21 @@ void wait_delay(uint64_t number_of_clks)
 	while ((platform_timer_get(timer) - current) < number_of_clks)
 		idelay(PLATFORM_DEFAULT_DELAY);
 }
+
+void wait_delay_ms(uint64_t ms)
+{
+	struct timer *timer = timer_get();
```

Comment:
How about calling wait_delay() with the result of clock_xx_to_ticks() so that implementation details like use of idelay with platform_default_delay are not duplicated? (would need to change many times in case of delay reimplementation)